src/main/java/org/entando/kubernetes/model/bundle/installable/PageModelInstallable.java

```diff
@@ -0,0 +1,37 @@
+package org.entando.kubernetes.model.bundle.installable;
+
+import java.util.concurrent.CompletableFuture;
+import lombok.extern.slf4j.Slf4j;
+import org.entando.kubernetes.model.bundle.descriptor.PageModelDescriptor;
+import org.entando.kubernetes.model.digitalexchange.ComponentType;
+import org.entando.kubernetes.service.digitalexchange.entandocore.EntandoCoreService;
+
+@Slf4j
+public class PageModelInstallable extends Installable<PageModelDescriptor> {
+
+    private EntandoCoreService engineService;
+
+    public PageModelInstallable(EntandoCoreService engineService, PageModelDescriptor pageModelDescriptor) {
+        super(pageModelDescriptor);
+        this.engineService = engineService;
+    }
+
+    @Override
+    public CompletableFuture install() {
```

Comment:
We don't need to do it now 

These implementations are all just about identical. Wonder if there is a way to pass the CompleteableFuture (or a wrapper with name/code) as an argument instead of stamping out the boilerplate for every single time. This is where the functional languages excel ...